cefi/handler/capitalcom.py

```diff
@@ -49,19 +49,8 @@ def __init__(
         """
         super().__init__(**kwargs)
         try:
-            if self.testmode:
-                self.client = DemoClient(
-                    log=self.user_id,
-                    pas=self.password,
-                    api_key=self.api_key,
-                )
-            else:
-                self.client = Client(
-                    log=self.user_id,
-                    pas=self.password,
-                    api_key=self.api_key,
-                )
 
+            self._build_client()
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **issue (performance):** Repeatedly calling `_build_client` in `get_quote` and `get_bid` may lead to performance issues.

Consider optimizing the client initialization to avoid potential overhead from reinitializing the client on each call.